mongodbatlas/resource_mongodbatlas_auditing.go

```diff
@@ -0,0 +1,132 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/mwielbut/pointy"
+
+	matlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+)
+
+const (
+	errorAuditingCreate = "error create MongoDB Auditing (%s): %s"
+	errorAuditingUpdate = "error update MongoDB Auditing (%s): %s"
+	errorAuditingRead   = "error reading MongoDB Auditing (%s): %s"
+	// errorAuditingDelete = "error delete MongoDB Auditing (%s): %s"
+)
+
+func resourceMongoDBAtlasAuditing() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceMongoDBAtlasAuditingCreate,
+		Read:   resourceMongoDBAtlasAuditingRead,
+		Update: resourceMongoDBAtlasAuditingUpdate,
+		Delete: resourceMongoDBAtlasAuditingDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"audit_authorization_success": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
```

Comment:
Are we using `Computed` to force the default value here? I think `audit_authorization_success` should be defaulted to `false` instead.

Reply:
No, the computed is to specify a value that not is set by the user. It means that it will set when the read method gets the value after de create the resource